### PR TITLE
Fix/ephemeral ports

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

File: packages/binding-coap/test/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
-        "rootDir": ".."
+        "rootDir": "..",
+        "experimentalDecorators": true
     },
-    "include": ["*.ts", "**/*.ts", "../src/**/*.ts"]
+    "include": ["test/*.ts", "test/**/*.ts"]
```
Comment (relu91) [inline packages/binding-coap/test/tsconfig.json:7]: Where are these changes coming from? are related to issue #599?

**Review by relu91 — APPROVED** [review]
I'm ok with merging, just a small question that needs to be answered before continuing.

File: packages/binding-coap/test/tsconfig.json (hunk 1 above)
Comment (erossignon) [reply packages/binding-coap/test/tsconfig.json:7]: This change doesn't belong here! let me remove it

**Comment by erossignon** [comment]
@relu91 , please merge this one in priority , it will fix master as well

**Comment by relu91** [comment]
Checked with daniel, merging.

**Pull request merged**